src/bp/ui-admin/src/Pages/Workspace/Bots.jsx

```diff
@@ -41,6 +42,22 @@ class Bots extends Component {
     this.setState({ isCreateBotModalOpen: !this.state.isCreateBotModalOpen })
   }
 
+  async exportBot(botId) {
```

Comment:
either extract this as a component or use a lib that does the same thing bc I know everywhere we'll do this, we'll ask why this hack exist.